### ISSUE TITLE
Segfault possibly related to gdatetime.c

### ISSUE THREAD

**Issue by crh23** [linuxmint/nemo#2166]
* nemo 4.0.6
 * Desktop nemo
 * Mint 19.1
 * `$ inxi -G`
`Graphics:  Device-1: NVIDIA GP104 [GeForce GTX 1070] driver: nvidia v: 418.56 
           Display: x11 server: X.Org 1.19.6 driver: nvidia resolution: 1920x1080~60Hz 
           OpenGL: renderer: GeForce GTX 1070/PCIe/SSE2 v: 4.6.0 NVIDIA 418.56`
 * 64 bit

**Issue**
Getting a segfault when some (but not all) terms in the search bar (the term that it happens for for me is `tf`). Happens in all three view configurations.
Debug output:
```$ gdb nemo
...
Reading symbols from nemo...Reading symbols from /usr/lib/debug/.build-id/70/4ebcac363b9867e69d29c4eb015853aff2adfd.debug...done.
done.
(gdb) r
Starting program: /usr/bin/nemo 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffe9a19700 (LWP 7692)]
...
Thread 1 "nemo" received signal SIGSEGV, Segmentation fault.
g_date_time_to_timezone (datetime=datetime@entry=0x0, tz=tz@entry=0x555555a9c780)
    at ../../../../glib/gdatetime.c:2637
2637	../../../../glib/gdatetime.c: No such file or directory.
```
**Steps to reproduce**
Not entirely sure. I can reproduce it consistently, but since it only happens on a subset of searches I don't know if it will replicate in another environment.

**Expected behaviour**
Not segfaulting

**Other information**
I've tried reinstalling nemo and libglib, no help.

**Comment by mtwebster** [comment]
I *think* this is fixed already for 4.2: https://github.com/linuxmint/nemo/commit/c4018c56d5405b2302b0d0e5

**Comment by crh23** [comment]
> I _think_ this is fixed already for 4.2: [c4018c5](https://github.com/linuxmint/nemo/commit/c4018c56d5405b2302b0d0e5)

It does appear to be, good work!
